### ISSUE TITLE
504 during large bot import 

### ISSUE THREAD

**Issue by frco9** [wit-ai/wit#729]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
bug

**What is the current behavior?**
504 error during bot import with a large set of entities

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Prepare a large set of entities (537) + expressions (10000) and try to import it. 
It return a 504 but the import job seems to continue in background as the bot is created with all entities. 
However, when I try to match a basic intent, nothing is returned so something may have got wrong during import :/ 
 
**What is the expected behavior?**

**What is the App ID where you are experiencing this issue (if applicable)?**
598d7ccc-6675-402e-a7c7-c75db9bc6de3

**Comment by hactar** [comment]
Looks like a duplicate of #513

**Comment by frco9** [comment]
Indeed, hope this will be fix soon 😞

**Comment by klemensz** [comment]
May be related to #683 and #642 as well. Basically, importing more than 1000 values for an entity leads to problems.

**Comment by stopachka** [comment]
Hey @klemensz 

Indeed, the import system still on old infra, and has scaling challenges. We are actively working on unblocking the infra for a system that would allow reliable imports and exports for large apps. In the meantime, I'd suggest stripping out the values, and using something like POST /samples in addition to import / export
